ide/ide.kit/nbproject/project.properties

```diff
@@ -50,5 +50,5 @@ test.jms.flags=\
  --add-opens=java.desktop/javax.swing.plaf.synth=ALL-UNNAMED \
  --add-opens=java.desktop/javax.swing.text=ALL-UNNAMED \
  --add-opens=java.desktop/sun.awt=ALL-UNNAMED \
- --add-modules=jdk.jdwp.agent,jdk.attach,jdk.jdi,jdk.jshell,java.compiler,jdk.compiler,jdk.management,jdk.unsupported,jdk.internal.le,jdk.internal.ed,jdk.internal.opt,jdk.internal.jvmstat \
+ --add-modules=java.compiler \
```

Comment:
JDK 23 added `jdk.internal.md` which we would have to add to the list. This would prevent the JVM from starting on JDKs <22.

Adding only `java.compiler` seems to work, since according to the doc this adds all transitive deps automatically (we don't have to list them apparently).